website/src/_includes/section-menu/contribute.html

```diff
@@ -12,20 +12,33 @@
 
 <li><span class="section-nav-list-main-title">Contribute</span></li>
 <li><a href="{{ site.baseurl }}/contribute/">Get started contributing</a></li>
-<li><a href="https://cwiki.apache.org/confluence/display/BEAM/Using+IntelliJ+IDE">Using IntelliJ IDE</a></li>
+<li><a href="https://cwiki.apache.org/confluence/display/BEAM/Contribution+FAQ">FAQ</a></li>
 <li>
   <span class="section-nav-list-title">Technical Docs</span>
 
   <ul class="section-nav-list">
+    <li><a href="{{ site.baseurl }}/roadmap/">Roadmap</a></li>
     <li><a href="{{ site.baseurl }}/contribute/testing/">Testing guide</a></li>
-    <ul>
-      <li><a href="{{ site.baseurl }}/contribute/precommit-triage-guide/">Pre-commit Slowness Triage Guide</a></li>
-    </ul>
+    <li><a href="{{ site.baseurl }}/contribute/precommit-triage-guide/">Pre-commit Slowness Triage</a></li>
     <li><a href="{{ site.baseurl }}/contribute/ptransform-style-guide/">PTransform style guide</a></li>
     <li><a href="{{ site.baseurl }}/contribute/runner-guide/">Runner authoring guide</a></li>
     <li><a href="{{ site.baseurl }}/contribute/docker-images/">Docker images</a></li>
     <li><a href="{{ site.baseurl }}/contribute/design-documents/">Design documents</a></li>
     <li><a href="{{ site.baseurl }}/contribute/dependencies/">Dependencies guide</a></li>
+    <li><a href="{{ site.baseurl }}/contribute/feature-branches/">Feature branches</a></li>
+  </ul>
+</li>
+<li>
+  <span class="section-nav-list-title">Tips</span>
+  <ul class="section-nav-list">
```

Comment:
I think it would be nice to create a dedicated "Tips" page on the Wiki and relocate all these links over there. This would remove clutter from the left nav and make the Wiki space more self contained (new pages can be added there over time, existing renamed etc.).

Reply:
Also, instead of "Tips" perhaps "Development Environment" would be more descriptive?

Reply:
Done. Changed the website sidebar to Development Environment. Organized the tips pages in the wiki under the FAQ which is renamed Development Environment FAQ.